metadata-ingestion/src/datahub/ingestion/source/fivetran/fivetran_log_api.py

```diff
@@ -208,50 +214,61 @@ def _get_jobs_list(
             )
         return jobs
 
-    def _get_user_email(self, user_id: Optional[str]) -> Optional[str]:
+    @functools.lru_cache()
+    def get_user_email(self, user_id: str) -> Optional[str]:
         if not user_id:
             return None
         user_details = self._query(
             self.fivetran_log_query.get_user_query(user_id=user_id)
```

Comment:
is this sending one query per user? ideally this would also do a bulk fetch

Reply:
Done